scripts/lib/google-sheets/incentives-sheet-exporter.ts

```diff
@@ -91,24 +92,62 @@ function createCellValue(
   return valToWrite;
 }
 
+function createCellDataValidation(
+  col: string,
+  fieldsWithEnums: FieldMetadata[],
+): sheets_v4.Schema$DataValidationRule | undefined {
+  for (const [ind, field] of fieldsWithEnums.entries()) {
+    if (col === field.column_aliases[0]) {
+      const columnLetter = colToLetter(ind * 3 + 1);
+      const lastEnumRow = Object.keys(field.values!).length + 1;
+      return {
+        strict: true,
+        // Make Google Sheets to show a dropdown UI element. Unfortunately,
+        // it's not possible in the API to force the "chip" version, just a
+        // regular list.
+        showCustomUi: true,
+        condition: {
+          type: 'ONE_OF_RANGE',
+          values: [
+            {
+              userEnteredValue: `='Standardized Enum List Values'!$${columnLetter}$2:$${columnLetter}$${lastEnumRow}`,
+            },
+          ],
```

Comment:
What if it's a potentially multi-valued column like payment methods?

Reply:
It's an awkward case in the current state, and this PR doesn't change anything about it. We currently have a `Multiple` type in the enums, so I propagated that into the spreadsheet-mappings.ts. But I don't like that because it's not a valid value in the schema.

The spreadsheet-to-json flow is capable of processing and validating comma-delimited arrays, so this is mostly about finding a way to let the spreadsheet allow that. For the least protection, we have the current approach, where collectors can go modify the allowed values. That could be our official guidance – if you need to add a multi-valued value, just go augment the spreadsheet.

Or instead of a "one of range" data validation, we could use a custom formula validation. Something like: `=iserror(product(map(split(Y9,","), lambda(txt, match(trim(txt),Y2:Y7,0)))))`, where Y9 is the cell in question, and Y2:Y7 is the allowable values. Basically splitting by comma and running each value through the allowable list, then verifying all of them are found. I'm going to add that to my handoff doc, but I'll try to get to it if I have time.